.gitignore

```diff
@@ -33,14 +32,10 @@ prerelease.txt
 # Build directory
 /build*
 emscripten_build/
-docs/_build
-docs/_static/robots.txt
-__pycache__
-docs/utils/*.pyc
-/deps/downloads/
-deps/install
-deps/cache
-cmake-build-*/
+/docs/_build
+/docs/_static/robots.txt
+/docs/utils/
```

Comment:
Did you see anything that might create this? If not we can just drop it.

Reply:
Yeah, `/docs/utils/` can probably be removed; `_build` and `_static` are generated via `spinx-build` (I guess) when `docs.sh` is run, so these would have to stay.

Reply:
https://github.com/ethereum/solidity/pull/13355#discussion_r940186007